deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/_endpoints.tpl

```diff
@@ -0,0 +1,2 @@
+{{- define "router.endpoint" }}{{ if .Values.grpc.routerHostname }}{{ .Values.grpc.routerHostname }}{{ else }}router.{{ .Values.global.baseDomain | required "grpc.routerHostname or global.baseDomain must be set"}}{{ end }}{{- end }}
```

Comment:
This is a failed attempt to create templates that I can use later in the other helm charts. I suspect it's only wrong when I make an invocation of them, so I'd rather keep them even if unused for now.